reference.md

```diff
@@ -161,7 +161,7 @@ admit lists, tuples only admit tuples, bools only admit bools.
 * A list type `A` admits another list type `B` iff `A.max-len >= B.max-len` and
   `A.entry-type` admits `B.entry-type`.
 * A buffer type `A` admits another buffer type `B` iff `A.max-len >= B.max-len`.
-* An optional type `A` admits another optional type `B` iff:
+* An optional type `A` admits another optional type `B` if:
```

Comment:
Is the thinking here that "iff" doesn't make sense since there is an "or"?

Reply:
I think the `iff` makes sense here but maybe it adds unnecessary complexity for people unfamiliar with it?
I'd say it can be merged 👍

Reply:
Maybe we should just expand all of the "iff"s to "if and only if"?